### ISSUE TITLE
How to track guest changes vmware-esxi6 template

### ISSUE THREAD

**Issue by roemer2201** [bonomani/devmon#21]
Dear All,

I would like to change the guest column from the vmware-esxi6 test so that it goes yellow if a change happens like, a vm is powered off, has [dis]appeared (probably vMotion). Can someone give me a hint, what transforms or thresholds can be used for this. The only thing I came up with is the "DELTA" transform, but I think it is only for numerical values. 

Thank you in advance!

Regards

**Comment by bonomani** [comment]
That's right, DELTA is for numericale value only and it is the only transform that have an history. But DELTA transform divide the result by the elapse of time: so this is not a real DELTA, it is a RATE........so probably the best thing is to create a new tramsform: for example "DIFF" inspire by DELTA that can accept string and compare the current value with the previous history... As I think it is a good idea I will try to implement it (if you did do it before me) so I keep the idea in my todo list,..

**Comment by roemer2201** [comment]
It is great to hear, that you will try to create a 'DIFF' transform. Unfortunately this is beyond my knowledge, I can only help when it comes to testing/debugging. Let me know, if I can help.

**Comment by bonomani** [comment]
Ideas: 
1. Changes that could/should be detected:
- Value of oids
- Index of oids
- amount of index of oids
- errors?
2. Adapt the delta transform ?
